worker/controlsocket/export_test.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package controlsocket
```

Comment:
We don't add these files any more. 

I generally advocate just testing in the same package. If you must have a separate test package, _package_test.go_ should always still go in the runtime package. Put these there instead.